modules/installation-creating-gcp-control-plane.adoc

```diff
@@ -57,29 +57,27 @@ resources:
   type: 05_control_plane.py
   properties:
     infra_id: '${INFRA_ID}' <1>
-    region: '${REGION}' <2>
-    zones: <3>
+    zones: <2>
     - '${ZONE_0}'
     - '${ZONE_1}'
     - '${ZONE_2}'
 
-    control_subnet: '${CONTROL_SUBNET}' <4>
-    image: '${CLUSTER_IMAGE}' <5>
-    machine_type: 'n1-standard-4' <6>
+    control_subnet: '${CONTROL_SUBNET}' <3>
+    image: '${CLUSTER_IMAGE}' <4>
+    machine_type: 'n1-standard-4' <5>
     root_volume_size: '128'
-    service_account_email: '${MASTER_SERVICE_ACCOUNT_EMAIL}' <7>
+    service_account_email: '${MASTER_SERVICE_ACCOUNT_EMAIL}' <6>
 
-    ignition: '${MASTER_IGNITION}' <8>
+    ignition: '${MASTER_IGNITION}' <7>
 EOF
 ----
 <1> `infra_id` is the `INFRA_ID` infrastructure name from the extraction step.
-<2> `region` is the region to deploy the cluster into, for example `us-central1`.
-<3> `zones` are the zones to deploy the control plane instances into, for example `us-central1-a`, `us-central1-b`, and `us-central1-c`.
-<4> `control_subnet` is the `selfLink` URL to the control subnet.
-<5> `image` is the `selfLink` URL to the {op-system} image.
-<6> `machine_type` is the machine type of the instance, for example `n1-standard-4`.
-<7> `service_account_email` is the email address for the master service account created above.
-<8> `ignition` is the contents of the `master.ign` file.
+<2> `zones` are the zones to deploy the control plane instances into, for example `us-central1-a`, `us-central1-b`, and `us-central1-c`.
+<3> `control_subnet` is the `selfLink` URL to the control subnet.
+<4> `image` is the `selfLink` URL to the {op-system} image.
+<5> `machine_type` is the machine type of the instance, for example `n1-standard-4`.
+<6> `service_account_email` is the email address for the master service account created above.
```

Comment:
>! above adj
Do not use to indicate a relative location in a document, as in “the above restrictions.” Use “previous” or
“preceding.”

I know that call-out wasn't added in this PR, but wanted to mention it.

Reply:
Might as well fix them when they're spotted, though. Thanks!